backend/integrations/email/email.go

```diff
@@ -36,15 +43,21 @@ func NewResendClient(settings config.ResendSettings, dev bool) EmailClientInterf
 	}
 }
 
-func send(email, subject, html string, client *resend.Client) error {
+func send(email string, subject string, template templ.Component, client *resend.Client) error {
+	buffer := new(bytes.Buffer)
+	err := template.Render(context.Background(), buffer)
+	if err != nil {
+		return fmt.Errorf("failed to render email template: %w", err)
+	}
+
 	params := &resend.SendEmailRequest{
-		From:    "onboarding" + constants.DOMAIN,
```

Comment:
A fault on my part but something I just noticed + can be done in a separate issue is that emails shouldn't all come from onboarding, should be dependent on scenario like password reset should come from a more specific email like a noreply

Reply:
resolved this